src/HTMLImports/importer.js

```diff
@@ -128,7 +132,7 @@ function makeDocument(resource, url) {
   var base = doc.createElement('base');
   base.setAttribute('href', url);
   // add baseURI support to browsers (IE) that lack it.
-  if (!doc.baseURI) {
+  if (!hasBaseURIAccessor(doc)) {
```

Comment:
I wonder if we should keep both checks here? In other words:
```js
  if (!doc.baseURI && !Object.hasOwnProperty(doc, 'baseURI')) {
```

(also can use hasOwnProperty instead of getOwnPropertyDescriptor, since we just want a boolean result)

Otherwise I worry this might send correct browsers down the wrong path. baseURI should never be an own-property of `document`... I guess old Safari was broken. According to the spec, it should be on Node.prototype: https://dom.spec.whatwg.org/#interface-node